src/EncryptionBehavior.php

```diff
@@ -169,34 +187,37 @@ public static function insertSearchableEncryptedColumnName(&$script, $realColumn
     }
 
     /**
-     * @return string[]
+     * @return array<mixed>
      */
     protected function getColumnNames(): array
     {
         return $this->getParameterValuesByPrefix('column_name');
     }
 
     /**
-     * @return string[]
+     * @return array<mixed>
      */
     protected function getSearchableColumnNames(): array
     {
-        return $this->getParameterValuesByPrefix('column_name_searchable');
+        return $this->getParameterValuesByPrefix('searchable_column_name');
     }
 
     /**
      * @param string $prefix
      *
-     * @return array
+     * @return array<mixed>
      */
     protected function getParameterValuesByPrefix(string $prefix): array
     {
         $parameterValues = [];
         foreach ($this->getParameters() as $parameterName => $parameterValue) {
-            if (strpos($parameterName, $prefix) !== false && !empty($parameterValue)) {
+            $prefixPosition = strpos($parameterName, $prefix);
+
+            if ($prefixPosition !== false && $prefixPosition === 0 && !empty($parameterValue)) {
```

Comment:
!empty($parameterValue) triggers CS failure, since it is cloaking for no reason.